components/Base/InputText/index.vue

```diff
@@ -0,0 +1,198 @@
+<template>
+  <div class="input-text">
+    <label
+      :class="{
+        'input-text__label' : true,
+        'input-text__label--italic': labelItalic,
+        'input-text__label--error' : error,
+      }"
+      :for="label"
+    >
+      {{ label }}
+    </label>
+    <div
+      :class="{
+        'input-text__wrapper-input': true,
+        'input-text__wrapper-input--focus': isFocused,
+        'input-text__wrapper-input--error': error
+      }"
+    >
+      <div
+        v-if="$slots['icon-left']"
+        :class="{
+          'input-text__icon-left': true,
+          'input-text__icon-left--border': iconLeftType === 'border' ? true : false,
```

Comment:
`input-text__icon-left--border': iconLeftType === 'border'` should be enough right ?

Reply:
ah thanks you i forgot that sorry my bad